kvakk_git_tools/ssb_gitconfig.py

```diff
@@ -208,22 +211,22 @@ def set_base_config(pl: Platform, test: bool) -> str:
         The recommended .gitattributes
     """
     temp_dir = Path.home() / "temp-ssb-gitconfig"
-    config_dir = temp_dir / "kvakk-git-tools" / "recommended"
+    config_dir = temp_dir / "kvakk-git-tools" / "kvakk_git_tools" / "recommended"
     dst = Path().home() / ".gitconfig"
     src = config_dir / f"gitconfig-{pl.name().value}"
     if test:
         src = config_dir / "gitconfig-dapla"
 
-    options = []
+    options = ["--branch", "2.0.0"]
```

Comment:
@Andilun The 2.0.0-branch will be removed when merging to main, right? Then you need to clear this option.

Reply:
2.0.0 is a tag.

`git clone https://github.com/statisticsnorway/kvakk-git-tools.git --branch 2.0.0`

Clones the 2.0.0 tag of the repo. This version locks the script such that future changes to project structure wont brake the script again.